core/src/com/threecubed/auber/entities/Player.java

```diff
@@ -39,6 +42,14 @@
   public boolean slowed = false;
   public boolean blinded = false;
 
+  public boolean invincible = false;
+  public boolean invisible = false;
+  public boolean superspeed = false;
+  public boolean vision = false;
+  public boolean insta_beam = false;
+
```

Comment:
These should probably be documented

Reply:
[Added](https://github.com/2020-eng1-team2/auber-pt2/commit/c9d5537527497b73cb366a588235802527d8a1d1)